### PR TITLE
Users/huliu/oc visitor integer literal parsing

### PR DESCRIPTION
**Pull request by huinanliu**
Add integer literal parsing logic in CypherVisitor.cs

### CODE REVIEW

File: src/openCypherParser/AST/CypherVisitor.cs [hunk 1]
```diff
@@ -530,6 +530,66 @@ private void UpdateEntityTypesInExpressionInPlace(IEnumerable<QueryExpression> e
             });
         }
 
+        // the order of parsing types referred from following:
```
Comment (direbearform) [inline src/openCypherParser/AST/CypherVisitor.cs]: Please use standard comment format (///) or move the comment inside the function

Comment (chenjerryliang) [inline src/openCypherParser/AST/CypherVisitor.cs]: Please use standard comment format (///) or move the comment inside the function definition.

File: src/openCypherParser/AST/CypherVisitor.cs [hunk 2]
```diff
@@ -530,6 +530,66 @@ private void UpdateEntityTypesInExpressionInPlace(IEnumerable<QueryExpression> e
...
+            else if (ulong.TryParse(literal, out ulong ulongValue))
+            {
+                return new QueryExpressionValue()
+                {
+                    Value = ulongValue
+                };
+            }
+            else
+            {
+                throw new TranspilerNotSupportedException($"Integer literal :{literal} out of supported range");
```
Comment (chenjerryliang) [inline src/openCypherParser/AST/CypherVisitor.cs]: I think in the exception message, we append 'is not supported'. so you only need to specific 'what' is not supported in the message parameter.

File: src/openCypherParser/AST/CypherVisitor.cs (hunk 1 above)
Comment (huinanliu) [reply src/openCypherParser/AST/CypherVisitor.cs]: updated by moving inside.

File: src/openCypherParser/AST/CypherVisitor.cs (hunk 2 above)
Comment (huinanliu) [reply src/openCypherParser/AST/CypherVisitor.cs]: "out of supported range" was removed. Thanks!

**Pull request merged**